### ISSUE TITLE
bigquery can't have dashes in col names

### ISSUE THREAD

**Issue by seamusabshere** [faradayio/dbcrossbar#50]
> A column name must contain only letters (a-z, A-Z), numbers (0-9), or underscores (_), and it must start with a letter or underscore.

https://cloud.google.com/bigquery/docs/schemas

```
BigQuery error in load operation: Invalid field name "amazon-order-id". Fields
must contain only letters, numbers, and underscores, start with a letter or
underscore, and be at most 128 characters long.
Error: `bq load` failed with exit code: 1 
```

**Comment by emk** [comment]
This wasn't completely fixed by PR #51, only partially fixed.

**Comment by emk** [comment]
Fixed in v0.2.2.
